src/api/services/auth.py

```diff
@@ -84,8 +84,10 @@ def whoami(self, context: Context):
 
       if user and context.is_admin_site and not(user.is_super_admin or user.is_community_admin):
         raise PermissionError
-
-      return serialize(user, full=True), None
+      #print(user)
+      s = serialize(user, full=True)
+      print(s)
+      return s, None
```

Comment:
Usually don't commit the print statements (except those which are helpful going forward)